### DISCUSSION TITLE
Is it possible to unrevoke a client ovpn profile ?

### DISCUSSION THREAD

**Discussion by ghost** [pivpn/pivpn#1428] [Support] [answered]
Hello,

Everything is in the title, is there a solution to unrevoke a client ?
I can clearly see my revoked client with the command pivpn -l, but can I easely unrevoke it ?

Thanks,
Sébastien

**Comment by coolapso** [comment, marked as answer]
Hi @Sebastien-Brach you don't unrevoke certificates, instead you generate a new one with the same name.

**Reply by lpoels** [reply]
Will this method generate a new key different from the original? 
i.e. Will have to replace the clients ovpn profile? or will the original continue to work?
